scripts/harmonize.py

```diff
@@ -189,7 +192,7 @@ def harmonize(file_in, file_ref, chr_col, pos_col, ref_col, alt_col, af_col, bet
             fcs = []
             for r in ref_vars:
```

Comment:
we can't have multiple hits to reference assuming gnomad does not have duplicates?

Also here we should not anymore do allele flipping as we have aligned to ref and gnomad is in ref /alt

Reply:
i.e dont equalize_to but just check if they are the same

Reply:
Before the added reference alignment during liftover, we checked by flipping what is the closest match in gnomad by af, and then we had to list all gnomad variants in the same position to find the closest match. If we can assume the variants are aligned, we can of course simplify this, good point!

In cases where we happen to already have the variants in build 38 but are not sure if they are aligned to reference, we might miss this functionality. But I guess that would be better served as a separate script.

Reply:
oh yea that case indeed if we flip to multiple gnomad variants so thats fine.
Not going to change much but when aligning all sumstats we probably should not flip anything during the run -> if not directly matching then not same variant. We don't need to take that in this PR though

Reply:
It's simple to change this to exact matching, see the edit I made: https://github.com/FINNGEN/META_ANALYSIS/pull/46/commits/c9f634b0a18e2c00dc2b2a5d1dfc09251d622d25.

I'm thinking here if we should add a flag to control whether to try matching with flipping or not. Could set the perfect matching as default so it would be more transparent in most of our cases. Just in case if we might need this functionality at some point..

Reply:
yea we could have a flag like `--pre_aligned` or the opposite of that to enable/disable flipping

Reply:
Added the `--pre_aligned` flag.